geomesa-gt/geomesa-gt-partitioning/src/test/scala/org/locationtech/geomesa/gt/partition/postgis/PartitionedPostgisDataStoreTest.scala

```diff
@@ -109,14 +109,37 @@ class PartitionedPostgisDataStoreTest extends Specification with BeforeAfterAll
   }
 
   "PartitionedPostgisDataStore" should {
+    "not work" in {
+      val ds = DataStoreFinder.getDataStore(params.asJava)
+      ds must not(beNull)
+
+      try {
+        ds must beAnInstanceOf[JDBCDataStore]
+
+        // This sft name exceeds 31 characters, so it should fail
+        val sft = SimpleFeatureTypes.renameSft(this.sft, "abcdefghijklmnopqrstuvwxyzabcde_____")
+        ds.getTypeNames.toSeq must not(contain(sft.getTypeName))
+        ds.createSchema(sft) must throwAn[java.io.IOException].like {
+          case e => e.getMessage mustEqual "Error occurred creating table"
+        }
+      } catch {
+        case NonFatal(e) => logger.error("", e); ko
+      } finally {
+        ds.dispose()
+      }
+      ok
+    }
+
     "work" in {
       val ds = DataStoreFinder.getDataStore(params.asJava)
       ds must not(beNull)
 
       try {
         ds must beAnInstanceOf[JDBCDataStore]
 
-        foreach(Seq("test", "test-dash")) { name =>
+        val sftNames: Seq[String] = Seq("abcdefghijklmnopqrstuvwxyzabcde")
```

Comment:
```suggestion
        val sftNames: Seq[String] = Seq("test", "test-thirty-chars-with-dashesz")
```
the dash in the `test-dash` actually was a bug at one point, so i'd like to keep it there.

Reply:
I agree - we can keep the dashes. But I think we'd want to test an sft name with 31 chars (rather than 30), since that is the maximum.